src/views/product-integration/index.tsx

```diff
@@ -69,54 +74,70 @@ export default function ProductIntegrationLanding({
 		})
 	}
 
+	// Generate Sidebar Information
+	const currentProduct = useCurrentProduct()
+	const sidebarNavDataLevels = [
+		generateTopLevelSidebarNavData(currentProduct.name),
+		generateProductLandingSidebarNavData(currentProduct),
+		{
+			backToLinkProps: {
+				text: `${currentProduct.name} Integrations Home`,
+				href: `/${currentProduct.slug}/integrations`,
+			},
+			levelButtonProps: {
+				levelUpButtonText: `${currentProduct.name} Home`,
+				levelDownButtonText: 'Previous',
+			},
```

Comment:
Feels like there's a TODO (not in the scope of this PR) here to have `levelUpButtonText` to go to **"[Product] Integrations Home"** (instead of [Product] Home), is that right?

Reply:
I think so; I'll add a TODO!

Reply:
Good callout!